ipv-stub-template.yml

```diff
@@ -124,7 +124,7 @@ Resources:
       Environment:
         Variables:
           ENVIRONMENT: !Sub ${Environment}
-          IPV_TOKEN_PUBLIC_SIGNING_KEY: ""
+          IPV_TOKEN_PUBLIC_SIGNING_KEY: "{{resolve:ssm:/orch-stubs/ipv-public-signing-key}}"
```

Comment:
Think its correct but just checking that the quotation marks are needed

Reply:
Yeah they are